src/main/java/utils/RestUtils/Gson.java

```diff
@@ -0,0 +1,11 @@
+package utils.RestUtils;
+
```

Comment:
Ces if you want to contribute to RestUtils keep on but with a different brach name